lumigator/python/mzai/backend/tests/conftest.py

```diff
@@ -28,7 +28,8 @@ def postgres_container():
 @pytest.fixture(scope="session", autouse=True)
 def db_engine(postgres_container: PostgresContainer):
     """Initialize a DB engine bound to the Postres container, and create tables."""
-    engine = create_engine(postgres_container.get_connection_url(), echo=True)
+    # engine = create_engine(postgres_container.get_connection_url(), echo=True)
```

Comment:
Why leave it commented?  I am personally in favor of removing it from the code if we're not using it (we have git history if we need this again)

Reply:
Ok, removing